src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -91,11 +91,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```

Comment:
Is there a specific reason for text vs. userText or is it just for clarity/consistency?

Reply:
It was the way the project was set up with that variable, I think as long as it's consistent it's good :)